evap/student/views.py

```diff
@@ -105,12 +105,51 @@ def sorter(evaluation):
         )
 
     unfinished_evaluations.sort(key=sorter)
+    info = Evaluation.annotate_with_participant_and_voter_counts(
```

Comment:
Can probably be unpacked by using `values_list` or something, so we immediatle have two meaningfully named local variables